examples/python/tools/trafilatura_example.py

```
@@ -0,0 +1,266 @@
+        if isinstance(result, dict) and 'error' in result:
+            print(f"✓ Correctly rejected: {unsafe_url}")
+        else:
+            print(f"✗ Failed to reject: {unsafe_url}")
+
+async def main():
+    """Run all examples"""
+    print("=== Custom Trafilatura Tools Examples ===\n")
+    
+    try:
+        # First check if we can create the tools
+        tools_instance = get_tools()
```

Ruff (0.11.9)

Local variable `tools_instance` is assigned to but never used

Remove assignment to unused variable `tools_instance`

(F841)